Common/include/element_structure.hpp

```diff
@@ -114,12 +134,6 @@ class CElement {
 	 */
 	unsigned short GetnGaussPoints(void);
 
-	/*!
```

Comment:
@rsanfer I noticed that everything dealing with pressure subintegration was not being used so I removed it.
I guessed that the initial idea was to have dedicated per-element methods, but eventually was replaced by instantiating lower order elements in the structural solver to deal specifically with the pressure term?
If this was not the case I can put it back.